options.js

```diff
@@ -77,15 +86,21 @@ function showNotifications(enabled, useHttps)
 
 function showHighlightUsers(groups)
```

Comment:
This fixes the bug where loadOptions (the old reset button) would add more Highlight Users groups every time you clicked. Even though the old reset button is gone, we'll need loadOptions for the new one (once implemented).